core/src/test/java/org/apache/iceberg/TestRewriteFiles.java

```diff
@@ -50,7 +50,9 @@ protected static List<Object> parameters() {
         new Object[] {1, "main"},
         new Object[] {1, "testBranch"},
         new Object[] {2, "main"},
-        new Object[] {2, "testBranch"});
+        new Object[] {2, "testBranch"},
```

Comment:
:| I still dislike how we have to do this for ParameterizedTestExtension and can't wait till we get first class support for class parameters. 

That said, we should probably change this to be generated using our `TestHelpers.ALL_VERSIONS` parameter so we don't miss this again in the future.

Reply:
Sure thing, fixed to use `ALL_VERSIONS`!